executor/services/api_client.py

```
@@ -0,0 +1,404 @@
+                f"preload_skills={data.get('preload_skills', [])}"
+            )
+            return TaskSkillsInfo(
+                task_id=data.get("task_id", int(task_id) if str(task_id).isdigit() else -1),
+                team_id=data.get("team_id"),
+                team_namespace=data.get("team_namespace", "default"),
+                skills=data.get("skills", []),
+                preload_skills=data.get("preload_skills", []),
+            )
+        else:
+            logger.warning(f"[fetch_task_skills] API returned no response for task {task_id}")
+        return default_result
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)